PRS6001-7000/PRS6820MaryMag.xml

```diff
@@ -45,18 +45,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
          <change who="PL" when="2019-03-27">removed occupation containing St, Saint, bibl, biblical or prophet, added keyword. See https://github.com/BetaMasaheft/Documentation/issues/1104</change>
+         <change when="2023-11-23" who="CH">updated names</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="2">
-               <persName>
-                  
-                  Mary Magdalene
-               </persName>
-               <persName type="alt">Maryam Mägdälawit</persName>
-               <!-- St -->
+               <persName xml:lang="en" xml:id="n1">Mary Magdalene</persName>
```

Comment:
I think it is best, when the English is the correspondence to Ethiopic and not a different name, to have it as one name; assigning the `type main` to the translation will push the English to be displayed first

```
               <persName xml:lang="gez" xml:id="n1">ማርያም፡ መግደላዊት፡</persName>
               <persName xml:lang="gez" corresp="#n1" type="normalized">Māryām Magdalāwit</persName>
               <persName xml:lang="en" corresp="#n1" type="main">Mary Magdalene</persName>
```

Reply:
Done.